src/ui/error/Error.tsx

```diff
@@ -0,0 +1,54 @@
+import { type FC } from 'react';
+
+import GiraffeEating from './assets/giraffe-eating.svg';
+import Giraffe from './assets/giraffe.svg';
+
+import css from './styles/styles.module.scss';
+
+import Button from '../button/Button';
+
+type Props = {
+  type: string;
+};
```

Comment:
Вот это сделай явным типом
```suggestion
type Props = {
  type: 'server' | 'client';
};
```

клиент просто для примера чтобы ты понимал. И вынеси его в `shared/types/ErrorType`